### PR TITLE
Auth intro tweaks

### PR DESCRIPTION
**Pull request by jpstroop**
Sorry @IIIF/editors, the intro to the auth spec was still feeling choppy to me. Here's an attempt to smooth it out.

### CODE REVIEW

**Comment by tomcrane** [comment]
:+1:

File: source/api/auth/0.9/index.md [hunk 1]
```diff
@@ -44,13 +44,14 @@ This is a work in progress. We are actively seeking implementations and feedback
...
+The IIIF (pronounced "Triple-Eye-Eff") specifications are designed to support uniform and rich access to resources hosted around the world. Open access to content is desirable, but internal policies, legal regulations, business models, and other constraints can require users to authenticate and be authorized to interact with some resources. The authentication process could range from a simple restriction by IP address or a click-through agreement, to a multi-factor scheme with a secure identity provider.
 
-Content providers often offer tiers of access beyond a simple all-or-nothing distinction. These tiers could provide versions of a resource that differ based on resolution, watermarking, or compression. Each version must have a distinct URI to prevent web caches from providing the wrong version.
+Content providers that need to restrict access to their resources may offer alternative versions that go beyond a simple all-or-nothing proposition, or have expressed a desire for a common pattern to facilitate this type of service. These alternative versions could be degraded based on resolution, watermarking, or compression, for example, but are often better than no access at all.
```
Comment (mikeapp) [inline source/api/auth/0.9/index.md]: "or have expressed..." - the "or" is not needed?

Comment (jpstroop) [inline source/api/auth/0.9/index.md]: "...may [already] offer...or have expressed a desire [but aren't doing it yet]..." is the distinction I was trying to surface. I see how you're reading it ('those who already offer this want to do it in the same way'), which is subtly different. Anyone else tripped up by this?

Comment (tomcrane) [inline source/api/auth/0.9/index.md]: I wasn't the first time round ( I read it your way) but I am being tripped by it now Mike has mentioned it!

Comment (zimeon) [inline source/api/auth/0.9/index.md]: Yes, this tripped me up. I think the whole "or expressed a desire .. service" clause is confusing, I suggest removing it entirely:

> Content providers that need to restrict access to their resources may offer alternative versions that go beyond a simple all-or-nothing proposition. These alternative versions could be degraded based on resolution, watermarking, or compression, for example, but are often better than no access at all.

Comment (tomcrane) [inline source/api/auth/0.9/index.md]: Content providers sometimes require more than simple all-or-nothing access. Common patterns are required to allow for alternative versions that differ in resolution, compression or by watermarking. For some types of content, degraded access is preferable to no access at all.

Too terse?

Comment (tomcrane) [inline source/api/auth/0.9/index.md]: Sorry @zimeon  - didn't see yours - pretty much the same!

Comment (jpstroop) [inline source/api/auth/0.9/index.md]: I'm trying to make it a little less terse. Watch this space.

File: source/api/auth/0.9/index.md [hunk 2]
```diff
@@ -44,13 +44,14 @@ This is a work in progress. We are actively seeking implementations and feedback
...
+The IIIF (pronounced "Triple-Eye-Eff") specifications are designed to support uniform and rich access to resources hosted around the world. Open access to content is desirable, but internal policies, legal regulations, business models, and other constraints can require users to authenticate and be authorized to interact with some resources. The authentication process could range from a simple restriction by IP address or a click-through agreement, to a multi-factor scheme with a secure identity provider.
 
-Content providers often offer tiers of access beyond a simple all-or-nothing distinction. These tiers could provide versions of a resource that differ based on resolution, watermarking, or compression. Each version must have a distinct URI to prevent web caches from providing the wrong version.
+Content providers that need to restrict access to their resources may offer alternative versions that go beyond a simple all-or-nothing proposition. These alternative versions could be degraded based on resolution, watermarking, or compression, for example, but are often better than no access at all.
```
Comment (jpstroop) [inline source/api/auth/0.9/index.md]: Went with @zimeon's suggestion. Was tempted to merge with the first paragraph, but left it alone.

**Comment by zimeon** [comment]
:+1:

**Comment by tomcrane** [comment]
:+1:

**Comment by mikeapp** [comment]
:+1:

File: source/api/auth/0.9/index.md (hunk 2 above)
Comment (azaroth42) [inline source/api/auth/0.9/index.md]: Don't we talk about tiers later on?  (Can we deploy to iiif.io branch to read?)

Comment (azaroth42) [inline source/api/auth/0.9/index.md]: I think the use of tiers in the intro sets us up for this section:  http://iiif.io/api/auth/0.9/#tiered-access
I prefer the original for that reason.

Comment (jpstroop) [inline source/api/auth/0.9/index.md]: I wasn't sure that "tiered" was readily understood in the intro, but maybe using this language to introduce the concept might work.

File: source/api/auth/0.9/index.md [hunk 3]
```diff
@@ -44,13 +44,14 @@ This is a work in progress. We are actively seeking implementations and feedback
...
+The IIIF (pronounced "Triple-Eye-Eff") specifications are designed to support uniform and rich access to resources hosted around the world. Open access to content is desirable, but internal policies, legal regulations, business models, and other constraints can require users to authenticate and be authorized to interact with some resources. The authentication process could range from a simple restriction by IP address or a click-through agreement, to a multi-factor scheme with a secure identity provider.
 
-Content providers often offer tiers of access beyond a simple all-or-nothing distinction. These tiers could provide versions of a resource that differ based on resolution, watermarking, or compression. Each version must have a distinct URI to prevent web caches from providing the wrong version.
+Content providers that need to restrict access to their resources may offer tiered access to alternative versions that go beyond a simple all-or-nothing proposition. These alternative versions could be degraded based on resolution, watermarking, or compression, for example, but are often better than no access at all.
```
Comment (jpstroop) [inline source/api/auth/0.9/index.md]: @azaroth42 :arrow_up:

File: source/api/auth/0.9/index.md (hunk 2 above)
Comment (azaroth42) [inline source/api/auth/0.9/index.md]: For the last sentence, how about:
> These alternatives could be degraded based on different tiers of access, where lower resolution, higher compression or watermarked versions are often preferable to no access at all.

File: source/api/auth/0.9/index.md (hunk 3 above)
Comment (azaroth42) [inline source/api/auth/0.9/index.md]: Looks good to me! :+1:

**Comment by azaroth42** [comment]
:boom: :gun:

**Pull request merged**